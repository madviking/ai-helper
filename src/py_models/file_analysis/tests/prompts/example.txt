Please analyze this file and extract its text content and provide a summary of its main content and purpose.