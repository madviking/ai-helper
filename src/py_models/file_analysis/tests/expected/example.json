{
    "text_content": "Sample extracted text from the analyzed file",
    "content_summary": "This file contains example content for testing file analysis functionality"
}